packages/core/src/client/bundlerClient.ts

```diff
@@ -21,35 +22,52 @@ import {
 } from "./decorators/bundlerClient.js";
 
 //#region BundlerClient
-export type BundlerClient<T extends Transport = Transport> = Client<
+export type BundlerClient<
+  T extends Transport = Transport,
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+> = Client<
   T,
   Chain,
   undefined,
-  [...PublicRpcSchema, ...BundlerRpcSchema],
-  PublicActions<T, Chain> & BundlerActions
+  [...PublicRpcSchema, ...BundlerRpcSchema<TEntryPointVersion>],
+  PublicActions<T, Chain> & BundlerActions<TEntryPointVersion>
 >;
 
 export const createBundlerClientFromExisting: <
-  T extends Transport | FallbackTransport = Transport
+  T extends Transport | FallbackTransport = Transport,
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
 >(
   client: PublicClient<T, Chain>
-) => BundlerClient<T> = <T extends Transport | FallbackTransport = Transport>(
+) => BundlerClient<T, TEntryPointVersion> = <
+  T extends Transport | FallbackTransport = Transport,
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+>(
   client: PublicClient<T, Chain>
-): BundlerClient<T> => {
-  return client.extend(bundlerActions);
+): BundlerClient<T, TEntryPointVersion> => {
+  return client.extend(bundlerActions<PublicClient, TEntryPointVersion>);
 };
 //#endregion BundlerClient
 
 /**
  * Creates a PublicClient with methods for calling Bundler RPC methods
  * @returns
  */
-export function createBundlerClient<TTransport extends Transport>(
-  args: PublicClientConfig<TTransport, Chain> & { type?: string }
-): BundlerClient<TTransport>;
+export function createBundlerClient<
+  TTransport extends Transport,
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+>(
+  args: PublicClientConfig<TTransport, Chain> & { type?: string } & {
+    entryPointVersion?: TEntryPointVersion;
+  }
+): BundlerClient<TTransport, TEntryPointVersion>;
 
-export function createBundlerClient(
-  args: PublicClientConfig & { type?: string }
+export function createBundlerClient<
+  TTransport extends Transport,
+  TEntryPointVersion extends EntryPointVersion = EntryPointVersion
+>(
+  args: PublicClientConfig<TTransport, Chain> & { type?: string } & {
+    entryPointVersion?: TEntryPointVersion;
+  }
```

Comment:
wait this shouldn't have changed, the overloaded definition was meant to be defined on the defaults as it was before